.github/workflows/sync.yml

```diff
@@ -0,0 +1,51 @@
+name: Daily list sync
+
+on:
+  schedule:
+    - cron: "30 15 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
+        with:
+          otp-version: "22.2"
+          elixir-version: "1.9.4"
+
+      - name: Download new list
+        run: curl https://raw.githubusercontent.com/wesbos/burner-email-providers/master/emails.txt -o priv/burner-email-providers/emails.txt
+
+      - name: Check for list changes
+        continue-on-error: true
+        id: changes
+        run: git update-index --refresh && git diff-index HEAD
+
+      - name: Commit changes
+        if: steps.changes.outcome != 'success'
+        run: |
+          git checkout main --
+          git config user.email "benjamin@opencollective.com"
+          git config user.name "Benjamin Piouffle"
+          git add priv/burner-email-providers/emails.txt
+
+      - name: Bump version
+        if: steps.changes.outcome != 'success'
+        run: |
+          CURRENT_VERSION=`cat VERSION`
+          NEW_PATCH=`echo $CURRENT_TAG | sed -E 's/v[0-9]+\.[0-9]+\.([0-9]+)/\1+1/' | bc`
+          NEW_TAG=`echo $CURRENT_TAG | sed -E "s/(v[0-9]+\.[0-9]+\.)[0-9]+/\1$NEW_PATCH/"`
+          echo $NEW_TAG > VERSION
+          git commit -m "chore: updated domain list"
+          git tag -a $NEW_TAG -m $NEW_TAG
+
+      - name: Push changes
+        if: steps.changes.outcome != 'success'
+        run: |
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
```

Comment:
Is this step really necessary? In a similar script I've been working on we managed to do this directly https://github.com/opencollective/opencollective-changelog/blob/main/.github/workflows/update-changelog.yml

Reply:
I've removed this line, but I'm not sure that it will work.